apps/linode-marketplace-influxdb/roles/influx/templates/docker-compose.yaml

```
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  influxdb:
+    image: influxdb:latest
+    ports:
+      - 8086:8086
```

```suggestion
      - 127.0.0.1:8086:8086
```
This should be the last commit, honestly. I noticed that this port was open and firewalls are not applied because docker manipulates that. Everything else looks good! Thanks Eric!